src/__test__/__snapshots__/countries.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../../app/store';
+import Countries from '../../pages/Countries';
+
+describe('display countries components', () => {
+  test('test if countries component renders in the dom', () => {
+    render(
+      <Provider store={store}>
+        <Countries />
+      </Provider>,
+    );
```

Comment:
- [ ]  Let's add more units tests for several components of the app and some pure functions in redux code.